docs/dockerhub.md

```diff
@@ -0,0 +1,78 @@
+# jpo-s3-deposit
+
+## GitHub Repository Link
+https://github.com/usdot-jpo-ode/jpo-s3-deposit
+
+## Purpose
+The purpose of the jpo-s3-deposit program is to deposit messages to an Amazon S3 bucket or some other destination.

```

Comment:
Maybe tweak a bit to something like:
`
The purpose of the jpo-s3-deposit program is to deposit messages to permanent storage, such as an Amazon S3 bucket, mongodb instance, or other similar system.
`